module/os/operation_siren.py

```diff
@@ -201,9 +217,11 @@ def os_meowfficer_farming(self, hazard_level=5, daily=False):
 
             self.globe_goto(zones[0])
             self.run_auto_search()
+            self.need_repair(self.config.ENABLE_OS_SETTING_REPAIR,
+                             self.config.OS_SETTING_REPAIR_THRESHOLD)
 
     def _clear_os_world(self):
-        for hazard_level in range(1, 7):
+        for hazard_level in range(self.config.OS_WORLD_MIN_LEVEL, self.config.OS_WORLD_MAX_LEVEL):
```

Comment:
Add 1 to OS_WORLD_MAX_LEVEL here, not in reading config

Reply:
No problem, easy adjustment